### PR TITLE
nuke: If normal reboot fails, power cycle

### PR DESCRIPTION
**Pull request by zmc**
I'm pretty surprised that we weren't doing this already.

### CODE REVIEW

**Comment by idryomov** [comment]
Could you please hold on on merging this until I try a few things?

File: teuthology/orchestra/remote.py [hunk 1]
```diff
@@ -444,6 +444,12 @@ def console(self):
             self._console = getRemoteConsole(self.name)
         return self._console
 
+    @property
+    def uptime(self):
+        proc = self.run(args=['cat', '/proc/uptime'], stdout=StringIO())
+        proc.wait()
+        return float(proc.stdout.read().split(' ')[0])
+
```
Comment (idryomov) [inline teuthology/orchestra/remote.py]: The whole "connect and then power cycle from the outside" seems wrong.  I've posted PR #1024 that should hopefully supersede this.

**Comment by dmick** [comment]
We decided against this in favor of @idryomov's solution, right?  Shall we close this if so?

**Comment by idryomov** [comment]
Yeah, I think we agreed on https://github.com/ceph/teuthology/pull/1024#issuecomment-277349956.  I have a couple of items on my TODO, but I haven't seen any new reports from @yuriw.

**Comment by dmick** [comment]
@zmc, do you agree this is dead?

**Comment by zmc** [comment]
I think so.

**Pull request closed without merging**